src/main/java/com/microsoftopentechnologies/windowsazurestorage/helper/AzureCredentials.java

```diff
@@ -182,6 +182,10 @@ public String getStorageAccountName() {
         return storageData.storageAccountName;
     }
 
+    public String getPlainStorageKey() {
+        return storageData.storageAccountKey.getPlainText();
```

Comment:
just didn't get how this change will make key in output get masked.

Reply:
output value is in plain text, so the credentials binding should in plain text too, otherwise cannot match when filtering the output